### ISSUE TITLE
[Bug]: Talkback Audio does not work fine

### ISSUE THREAD

**Issue by unstressable** [homebridge-eufy-security/plugin#196]
### What happened?

Hey there,

I use a T8200 Video doorbell and want to use talkback Audio.

The problem is, that there is just some fancy noise when I want to speak to the doorbell. Hearing is fine so far.
I'm using a Rapsi4, thats why I only can use less resolution.

### Device Type

Doorbell (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v16 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
nv
```

### config

```shell
{
      "platform": "EufySecurity",
      "username": "mail",
      "password": "pwd",
      "pollingIntervalMinutes": 30,
      "hkHome": 1,
      "hkAway": 0,
      "hkNight": 3,
      "hkOff": 63,
      "enableDetailedLogging": 0,
      "country": "DE",
      "CameraMaxLivestreamDuration": 600,
      "cameras": [
        {
          "serialNumber": "ID",
          "enableCamera": "true",
          "enableButton": false,
          "motionButton": false,
          "unbridge": true,
          "forcerefreshsnap": true,
          "videoConfig": {
            "audio": true,
            "audioSampleRate": 0,
            "useSeparateProcesses": true,
            "maxStreams": 1,
            "maxWidth": 640,
            "maxHeight": 480,
            "maxBitrate": 100,
            "maxFPS": 15
          },
          "talkback": true
        }
```

**Comment by unstressable** [comment]
> Did you check this [Common Issues](https://github.com/homebridge-eufy-security/plugin/wiki/Common-Issues) pages ?

Hi, how r u? ;)

**Comment by thieren** [comment]
Hi @unstressable 

Is it possible to provide a short audio/video clip of the resulting talkback output?

**Comment by unstressable** [comment]
yep, here it is:
https://file.io/jEsURmqSn32t

**Comment by thieren** [comment]
@unstressable 

the file is flagged as 'deleted' for me. But never mind. I had some feedback from another user on discord already.
I'll be trying to see if I can fix this with him later this month. Stay tuned.

**Comment by unstressable** [comment]
sorry, idk why.
Here is another link:

https://mega.nz/file/iBBjDA7I#noGcTy3jzl195nq2yTabn2RQ0WT784oljYzHEy_cZqI

**Comment by thieren** [comment]
The issue is most likely that the channel configuration was wrong. The default was mono.

New since beta.14: You can select the talkback channel configuration in the device settings. Please update to stereo and see if that helps your issue.

**Comment by lenoxys** [comment]
**Thank you for taking the time to report this issue.**

As part of a repository cleanup, we are closing issues that have been open for more than 12 months without resolution. A **beta version** of the plugin has been released that includes significant changes and improvements.

We encourage you to try the latest beta and see if your issue has been addressed. If the problem persists, please feel free to open a new issue with updated details and logs from the latest version.

Thank you for your patience and support!
